.gitignore

```diff
@@ -55,6 +55,9 @@ mx.scratch
 # Covers JetBrains IDEs: IntelliJ, RubyMine, PhpStorm, AppCode, PyCharm, CLion, Android Studio and WebStorm
 # Reference: https://intellij-support.jetbrains.com/hc/en-us/articles/206544839
 
+# VSCode
+.vscode
```

Comment:
The .gitignore means that it will ignore any files inside the .vscode directory and you (or anyone else actually) don't have to worry about accidentally committing vscode files lol
- Note: this only works if the file isn't cached